README.md

```
@@ -542,10 +542,17 @@ Regeneration steps:
+   ```bash
+   pip install "numpy==1.26.4" "setuptools<63" "wheel"
+   ```
+
 3. From the repo root, run:
-   `python -m numpy.f2py -m libphsh -c phaseshifts/lib/libphsh.pyf phaseshifts/lib/libphsh.f`
-   `--build-dir /tmp/libphsh-f2py`
+
+   ```bash
+   python -m numpy.f2py -m libphsh -c phaseshifts/lib/libphsh.pyf \
+     phaseshifts/lib/libphsh.f --build-dir /tmp/libphsh-f2py
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)